Tests/TwoDimension/AreaTests.cs

```diff
@@ -156,6 +156,8 @@ public void PositionSet()
 			Assert.That(() => area.SetPosition(null, mockPosition.Object), Throws.ArgumentNullException);
 			Assert.That(() => area.SetPosition(mockLevel.Object, null), Throws.ArgumentNullException);
 
+			// TODO: Issue 13 (https://github.com/Wizcorp/TileSystem/issues/13)
```

Comment:
Yes I hope that each one will be a pull request when it's fixed, also the task manager in visual studio is pretty nice for this kind of thing.